wrappers/localsearch.rb

```diff
@@ -0,0 +1,28 @@
+# Copyright © Mapotempo, 2022
+#
+# This file is part of Mapotempo.
+#
+# Mapotempo is free software. You can redistribute it and/or
+# modify since you respect the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+#
+# Mapotempo is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the Licenses for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Mapotempo. If not, see:
+# <http://www.gnu.org/licenses/agpl.html>
+#
+require './wrappers/wrapper'
+require './wrappers/ortools_vrp_pb'
+require './wrappers/ortools_result_pb'
+module Wrappers
+  class Localsearch < Ortools
```

Comment:
I'm wondering about the name `Localsearch`. 

This class is called `Localsearch`, is used if `unconstrainted_initialization` is true and uses a `initial_solution`. 

Is there not a misunderstanding in the names?